appium-uiautomator2-server/build.gradle

```diff
@@ -37,3 +37,17 @@ dependencies {
     androidTestCompile 'com.android.support.test.espresso:espresso-core:2.0'
 
 }
+
+task installAUT(type: Exec) {
```

Comment:
needs to be like this:

```
task installAUT(type: Exec, dependsOn: 'installDebug') {
    group = 'test'
    description = 'Install app under test'
```

otherwise it won't get installed as part of the regular install debug flow.